pagopa/io-app
# chore: [IOCOM-1560] Fims history screen

## Short description
Addition of FIMS history screen, with email export coming soon

<img src="https://github.com/user-attachments/assets/c3d804a1-96f6-489a-9d2e-5d5227731af3" width=250/>


## List of changes proposed in this pull request
- required i18n keys
- new simplified utility pot fold function
- required selectors
- required screen and components
- full design-perfect screen coming soon due to problematic component interactions
- new auto fetching/refreshing service data hook

## How to test
using the dev-server, while checked out in the fims testing branch, navigate to profile>privacy>third party accesses and make sure everything is working as intended
